index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
+        integrity="sha384-xxzQGERXS00kBmZW/6qxqJPyxW3UR0BPsL4c8ILaIWXva5kFi7TxkIIaMiKtqV1Q" crossorigin="anonymous">
+    <link rel="shortcut icon" type="image/x-icon" href="img/favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="img/favicon.ico">
+    <title>Intuit Accounts</title>
+</head>
+
+<body>
+    <nav>
+        <div id="top-div">
+            <div id="A">
+                <div class="sing-in">
+                    <p>Already have an Intuit account?</p>
+                </div>
+                <a href="" class="id-pasword">I forgot my User ID or Password</a>
+            </div>
+            <div id="B">
+                <button id="button-nav" type="button">
+                    <img id="lock" src="img/lock.png" alt="lock">
+                    sign in
+                </button>
+            </div>
+        </div>
+    </nav>
+    <header>
+        <img src="img/logo-intuit.png" alt="Logo Intuit">
+    </header>
+    <main>
+        <header id="main-header">
+            <ul id="form-ul">
+                <li id="img1"></li>
+                <li>mint</li>
+                <li id="img2"></li>
+                <li>quickbooks</li>
+                <li id="img3"></li>
+                <li>turbotax</li>
+            </ul>
+        </header>
+        <form id="form" action="#">
+            <header id="form-header">
+                <h2>Create an Intuit account</h2>
+                <div id="form-header-div">
+                    <p class="p-1">One account for everything Intuit, including Mint</p>
+                    <a href="" class="p-1">Learn more</a>
+                </div>
+            </header>
+            <fieldset>
+                <label class="form-label" for="mail">Email address</label>
+                <input id="mail" name="e-mail" class="form-label ipt" type="email">
+            </fieldset>
+            <fieldset>
+                <label class="form-label" for="phone">Phone (recommended)</label>
+                <input id="phone" name="phone" class="form-label ipt" type="number">
```

Comment:
- [ ] Please use an [`<input>` of type tel](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) for this input field.